tests/functional/tests.py

```diff
@@ -68,31 +74,120 @@ def read(self, amt=64*1024):
         self._offset_location += len(data)
         return data
 
+def initialize_log_output(meth, alert = None):
```

Comment:
I think a better name would be `new_log_result()`, `initialize_log_output()` means like a one time action.

Reply:
Done.